### PR TITLE
Schedule import: Add overview

### PR DESCRIPTION
**Pull request by MyIgel**
Created an overview page from where multiple schedules can be imported

### CODE REVIEW

File: resources/views/admin/schedule/index.twig [hunk 1]
```diff
@@ -6,35 +6,56 @@
...
+        <h1>
+            {% block content_title %}{{ title }}{% endblock %}
 
-        {% for message in errors|default([]) %}
-            {{ m.alert(__(message), 'danger') }}
-        {% endfor %}
-        {% for message in success|default([]) %}
-            {{ m.alert(__(message), 'success') }}
-        {% endfor %}
+            {% if schedules|default([]) %}
```
Comment (msquare) [inline resources/views/admin/schedule/index.twig:12]: This shows the add button only if there is at least one schedule configured. The button should always be visible.

File: resources/views/admin/schedule/index.twig [hunk 2]
```diff
@@ -6,35 +6,56 @@
...
+                            <tbody>
+                            {% for schedule in schedules %}
+                                <tr>
+                                    <td>{{ schedule.name }}</td>
+                                    <td>{{ schedule.url }}</td>
+                                    <td>
+                                        <div class="btn-group">
+                                            <a
+                                                href="{{ url('/admin/schedule/load/' ~ schedule.id) }}"
+                                                class="btn btn-default">
```
Comment (msquare) [inline resources/views/admin/schedule/index.twig:43]: Please add btn-xs

**Review by msquare — CHANGES_REQUESTED** [review]
Works out of the box as well. Very nice!
Only some minor corrections needed. Thank you.

File: resources/views/admin/schedule/index.twig (hunk 1 above)
Comment (MyIgel) [reply resources/views/admin/schedule/index.twig:12]: Whops, äh, will fix that :D

**Review by msquare — APPROVED** [review]

**Pull request merged**